.github/workflows/test-e2e.yml

```diff
@@ -20,12 +20,6 @@ jobs:
       - uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 #v5.5.0
         with:
           go-version-file: "go.mod"
-      - name: Setup kubeconfig
```

Comment:
[question] Can we delete this file entirely? `test-docker-e2e` is triggered by `test.yml`